### PR TITLE
Restaurant-Website

### PR DESCRIPTION
**Pull request by emmanuellekamwa**
In this project I have used the knowledge of DOM manipulation and webpack to build this simple restaurant website.

### CODE REVIEW

File: src/scripts/contact.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+/* eslint-disable linebreak-style */
+/* eslint-disable quotes */
+/* eslint-disable linebreak-style */
+/* eslint-disable indent */
+const addContactsContent = () => {
+    const contactsContent = document.createElement("div");
+
+    contactsContent.classList.add("contacts-info-div");
+    contactsContent.innerHTML =
+        '<span><i class="fas fa-envelope"></i>emmanuellekamwa@gmail.com</span><span><i class="fab fa-twitter"></i>@AlixKamwa</span><span><i class="fab fa-github"></i>emmanuellekamwa</span>';
```
Comment (bellom) [inline src/scripts/contact.js]: We need to use `document.createElement` and `document.appendChild` to appends HTML elements.

File: src/scripts/contact.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable operator-linebreak */
+/* eslint-disable linebreak-style */
+/* eslint-disable quotes */
+/* eslint-disable linebreak-style */
+/* eslint-disable indent */
```
Comment (bellom) [inline src/scripts/contact.js]: Inline comment: 
With the ESLint inline comment, you are indirectly changing the linter's rules by disabling these rules, let's fix the codebase instead.

If you are using VSCode, you can delete the inline comment and hover over the error(configure Eslint checks locally). And you will see your options. Click on the more info button that will take you to a GitHub repo where it is explained why you are getting this error. I find this feature very educational. Please, try to learn and implement the necessary rules so you don't have to keep this disabling line on top of each file. I believe you can do this! 💪😇.

File: src/scripts/index.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable indent */
+/* eslint-disable linebreak-style */
+/* eslint-disable quotes */
```
Comment (bellom) [inline src/scripts/index.js]: Inline comment: 
With the ESLint inline comment, you are indirectly changing the linter's rules by disabling these rules, let's fix the codebase instead.

File: src/scripts/loadingpage.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable indent */
+/* eslint-disable linebreak-style */
+/* eslint-disable quotes */
+/* eslint-disable linebreak-style */
```
Comment (bellom) [inline src/scripts/loadingpage.js]: Inline comment: 
With the ESLint inline comment, you are indirectly changing the linter's rules by disabling these rules, let's fix the codebase instead.

File: src/scripts/menu.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable operator-linebreak */
+/* eslint-disable linebreak-style */
+/* eslint-disable quotes */
+/* eslint-disable linebreak-style */
+/* eslint-disable indent */
+/* eslint-disable linebreak-style */
```
Comment (bellom) [inline src/scripts/menu.js]: Inline comment: 
With the ESLint inline comment, you are indirectly changing the linter's rules by disabling these rules, let's fix the codebase instead.

If you are using VSCode, you can delete the inline comment and hover over the error(configure Eslint checks locally). And you will see your options. Click on the more info button that will take you to a GitHub repo where it is explained why you are getting this error. I find this feature very educational. Please, try to learn and implement the necessary rules so you don't have to keep this disabling line on top of each file. I believe you can do this! 💪😇

File: src/scripts/menu.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+const addMenuContent = () => {
+    const menuContent = document.createElement("div");
+    menuContent.classList.add("menu-info-div");
+    menuContent.innerHTML =
+        "<h1>EXOTIC MENU</h1><h2>Suchi Rolls</h2><ul><li>Maki Mix  <span>7,00£</span></li><li>Suchi Vegeterian <span>7,50£</span></li><li>Sashimi Zalm <span>6,00£</span></li>Urakami Box<li><span>18,00£</span></li></ul><h2>Exotic Fast Food</h2><ul><li>Döner Kebab (Chicken) <span>4,50£</span></li><li>Baguettes <span>3,50£</span></li><li>Currywurst<span>3,50£</span></li><li>Egg Rolls<span>3,00£</span></li><li>Chicken Nuggets<span>4,50£</span></li><li>Pizza <span>5,00£</span></li><li>Homemade Burger<span>5,00£</span></li></ul>";
```
Comment (bellom) [inline src/scripts/menu.js]: We need to use `document.createElement` and `document.appendChild` to appends HTML elements.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @emmanuellekamwa  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**1. This project comes after the introduction of the ES6 class keyword, ES6 modules, and Webpack. The main goal of this project is for you to show the understanding of the benefits of modularized code as well as the ability to set up a javascript application made of different modules using Webpack.**

_I can't run `npm run build` to generate `dist folder`, I should be able to run `npm run build` to generate `dist folder` at my end, currently using the one created by you, shouldn't be like that. kindly set up the Webpack properly. checkout [Webpack Setting](https://www.theodinproject.com/paths/full-stack-ruby-on-rails/courses/javascript/lessons/webpack)_

**2. Kindly add a live demo link to your README file.**

**3. Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by taiwo2 — APPROVED** [review]
# STATUS : APPROVED ✔👌🎉
@emmanuellekamwa,  Weldone and nice job for implementing suggested changes.
you can merge and prepare for your next  task

**Pull request merged**